zend_operator.go

```diff
@@ -0,0 +1,102 @@
+package gophplib
+
+import (
+	"database/sql"
+	"fmt"
+	"math"
+	"net"
+	"os"
+	"reflect"
+)
+
+type toStringAble interface {
+	toString() string
+}
+
+type Floats interface {
+	float32 | float64
+}
```

Comment:
Looking at this now, it seems like this part doesn't need to be a public interface of the library. Is there any particular reason why you exported it?